docs/basics/1d_hist.rst

```diff
@@ -265,3 +265,46 @@ String category
 .. image:: ../img/1d_str_category.svg
     :alt: String category plot
     :width: 500
+
+
+Using multiple histograms
+-------------------------
+
+With multiple histograms, the :func:`plot_hist() <plothist.plotters.plot_hist>` function will correctly put them side by side:
+
+.. image:: ../img/1d_side_by_side.svg
+    :alt: Side by side histograms
+    :width: 500
+
+.. literalinclude:: ../examples/1d_hist/1d_side_by_side.py
+    :language: python
+    :start-after: ###
+
+To also add the number of entries on top of each bar, you can use the following code:
```

Comment:
I would remove this, the doc should stay as simple as possible (compare it with the rest of the page). We could add a new example in the gallery with the numbers on top instead.